tests/core/arch-arm/stalker-arm.c

```diff
@@ -2505,6 +2506,15 @@ unfollow_during_transform (GumStalkerIterator * iterator,
   ctx->num_blocks_transformed++;
 }
 
+TESTCASE (follow_me_should_support_nullable_event_sink) {
```

Comment:
```suggestion
TESTCASE (follow_me_should_support_nullable_event_sink)
{
```

(Here and the in the corresponding code for the other archs.)